src/test/crimson/ctest_seastar.h

```diff
@@ -0,0 +1,76 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
the major part of this file was created by me. could please use my Signed-off-by ?